app/views/catalog/_facet_limit.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= render_facet_limit_list paginator, field_name %>
 
   <% unless paginator.last_page? || params[:action] == "facet" %>
-    <li class="more_facets_link"><%= link_to 'more ' + facet_field.label.pluralize + '»',
+    <li class="more_facets_link"><%= link_to 'more ' + facet_field_label(facet_field.field).pluralize + '»',
```

Comment:
"more" is hard coded. Should we switch to Blacklight current practice? https://github.com/projectblacklight/blacklight/blob/v6.9.0/app/views/catalog/_facet_limit.html.erb#L7